lib/modules/swarm/index.js

```diff
@@ -80,7 +80,12 @@ class Swarm {
       let currentPIterationVersion = require('../../../package.json').dependencies["p-iteration"];
       if (pIterationVersion !== currentPIterationVersion) {
         self.events.request("version:getPackageLocation", "p-iteration", pIterationVersion, function(err, location) {
-          self.embark.registerImportFile("p-iteration", fs.dappPath(location));
+          if(!err){
```

Comment:
Cleaner to check `if (err)` and then return to get rid of the else

Reply:
Fixed